app/models/concerns/startdate_beyond_osra_establishment_validation.rb

```diff
@@ -0,0 +1,12 @@
+module StartdateBeyondOsraEstablishmentValidation
+	include ValidatorHelpers
+
+	OSRA_ESTABLISHMENT_DATE = Date.new(2012,04,01)
```

Comment:
But then at least add a formal documentation about them. I don't want to be looking for them in many files, that's the purpose of the global shared location.